OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -71,9 +71,14 @@ public class MobileInfo : PausableConditionalTraitInfo, IMoveInfo, IPositionable
 		[Desc("Can move backward if possible")]
 		public readonly bool CanMoveBackward = false;
 
-		[Desc("After how many ticks the actor will turn forward during backoff")]
+		[Desc("After how many ticks the actor will turn forward during backoff.",
+		" Set it to -1 to ignore the time.")]
```

Comment:
```suggestion
		[Desc("For how much time (in ticks) should the unit move backwards?",
		" If set to -1 the unit will be allowed to drive backwards forever")]
```

I think the description could be improved while we are at it

Reply:
I think `forever` maybe a little confusing, which looks like it will also ignore `MaxBackwardCells`

Reply:
I don't find them confusing at all. It is explained in the first sentence what the trait limits, so the word forever is linked to time here.

Reply:
Changed to "If set to -1 the unit will be allowed to move backwards without time limit"